Composer/packages/extensions/visual-designer/src/editors/StepEditor.tsx

```diff
@@ -56,6 +57,17 @@ export const StepEditor = ({ id, data, onEvent, trigger }): JSX.Element => {
   const editorHeight = HeadSize.height + TailSize.height + contentBoundary.height;
   const editorAxisX = editorWidth / 2;
 
+  const { width } = useWindowDimensions();
+
+  useEffect(() => {
+    addCoachMarkRef({
+      action: {
+        x: (width + editorWidth) / 2,
+        y: (hasNoSteps ? InitNodeSize.height / 2 : (3 * InitNodeSize.height) / 2 + ElementInterval.y) + 48,
+      },
+    });
+  }, []);
```

Comment:
Should this have a dependency on width? What happens if a user resizes the window?

Reply:
Yup